docs/src/test/scala/docs/http/scaladsl/server/FullSpecs2TestKitExampleSpec.scala

```diff
@@ -53,8 +53,8 @@ class FullSpecs2TestKitExampleSpec extends Specification with Specs2RouteTest {
     "return a MethodNotAllowed error for PUT requests to the root path" in {
       // tests:
       Put() ~> Route.seal(smallRoute) ~> check {
-        status === StatusCodes.MethodNotAllowed
-        responseAs[String] === "HTTP method not allowed, supported methods: GET"
+        status shouldEqual StatusCodes.MethodNotAllowed
```

Comment:
My understanding is that `shouldEqual` compares against `Any` and `===` compares against a value of the same type. So my question is whether it is not better to keep `===` so that the test fails to compile if you compare for example status values with a string?